src/Components/Tab.tsx

```diff
@@ -0,0 +1,93 @@
+import headlessEngine from "../Engine";
+import {
+  buildTab,
+  TabProps,
+  Tab as HeadlessTab,
+  buildQueryExpression,
+  TabOptions,
+} from "@coveo/headless";
+import React from "react";
+import Tab from "@mui/material/Tab";
+export default class TabBar extends React.Component<TabProps> {
```

Comment:
I believe that a better name than "TabBar" would be "Tab". Right now, in App.tsx, it's a bit confusing to have:

```
Tabs
  - TabBar
  - TabBar
  - TabBar
```

I think it would be more intuitive to have the following, like in the Material UI docs:

```
Tabs
  - Tab
  - Tab
  - Tab
```